integration/localnet/Makefile

```diff
@@ -71,6 +73,13 @@ start:
 	DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker-compose -f docker-compose.nodes.yml build
```

Comment:
you can consider using `$(MAKE) start-thin` here

Reply:
(or possibly even renaming it so there is `start-with-metrics` and `start`.)

Reply:
I think we want to keep `start` as the main entrypoint but I can use `$(MAKE) start-thin` in start for reuse/clarity. Change incoming.